aws/resource_aws_waf_web_acl_test.go

```diff
@@ -423,6 +471,8 @@ resource "aws_waf_web_acl" "test" {
   default_action {
     type = "ALLOW"
   }
+
```

Comment:
Can you please remove these extraneous line changes? Thanks!

Reply:
done

Reply:
There's still a bunch of these -- lines 488, 502, 503, 546, 547, etc. If you're not using the GitHub UI, maybe they're not obvious, but we'd like to avoid file churn that could cause merge conflicts 😅